api-reference/10 UI Widgets/dxFilterBuilder/1 Configuration/onEditorPreparing.md

```diff
@@ -94,11 +95,12 @@ In the following code, a default editor is replaced with the DevExtreme [TextAre
     export class AppComponent {
         onEditorPreparing (e) { 
             if (e.dataField == "description") {
+                const defaultHandler = e.editorOptions.onValueChanged;
                 e.editorName = "dxTextArea"; 
                 e.editorOptions.showClearButton = true;
                 e.editorOptions.onValueChanged = (event) => {
                     let value = event.value;
-                    e.setValue(value.toLowerCase()); 
+                    defaultHandler(value.toLowerCase()); 
```

Comment:
It seems that this code doesn't work: https://codepen.io/romanbalaban/pen/PoZdxWE

Reply:
@vconst

Reply:
defaultHandler(event)